ci/gitlab-cscs.yml

```diff
@@ -87,6 +121,15 @@ multi node release:
     SLURM_NTASKS: 2
     ALLOCATION_NAME: arbor-ci-release-$CI_PIPELINE_ID
 
+test-ault-fujitsu:
+    needs: []
+    tags:
```

Comment:
Do we need: `only: ['master', 'staging', 'trying']` here?

Reply:
Would that mean you can't run a `bors try` on any PR?

Reply:
I think our PRs get tested on the `trying` branch.

Reply:
see https://bors.tech/documentation/getting-started/

Reply:
I was mostly wondering why the settings for `test_ault_fujitsu` are different from `single_node_release` and `multi_node_release`.

Reply:
The `test_ault_fuji` test is performed in a single stage by a runner that executes a script that configures, builds, then runs the test on an ARM node.
The tests on Daint are performed in stages, where Arbor is first built in a container, then later stages that allocate nodes and run the tests.
In an ideal world we could put the daint tests in one yml file, and the ault tests in another, because they are not related at all. But Gitlab runners force us to put all the tests in a single file (unlike GitHub Actions).

Reply:
Thanks for the explanation :)